.github/workflows/integration-test-DEV.yaml

```diff
@@ -19,12 +24,40 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+
+
+
 jobs:
+
+  check-config:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if DEV_REGULAR_USER_API_KEY is defined
+        run: |
+          if [[ -z "${{ secrets.DEV_REGULAR_USER_API_KEY }}" ]]; then
+            echo "Error: Missing secret: Please configure DEV_REGULAR_USER_API_KEY."
+            exit 1
+          fi
+      - name: Check if DEV_ADMIN_USER_API_KEY is defined
+        run: |
+          if [[ -z "${{ secrets.DEV_ADMIN_USER_API_KEY }}" ]]; then
+            echo "Error: Missing secret: Please configure DEV_ADMIN_USER_API_KEY."
+            exit 1
+          fi
+      - name: Check if IRS_CUCUMBER_PUBLISH_TOKEN is defined
+        run: |
+          if [[ -z "${{ secrets.IRS_CUCUMBER_PUBLISH_TOKEN }}" ]]; then
+            echo "Error: Missing secret: Please configure IRS_CUCUMBER_PUBLISH_TOKEN."
+            exit 1
+          fi
+        shell: bash
+
   trigger-integration-test:
+    needs: check-config
     uses: ./.github/workflows/cucumber-integration-test.yaml
     secrets:
       regularUserApiKey: ${{ secrets.DEV_REGULAR_USER_API_KEY }}
       adminUserApiKey: ${{ secrets.DEV_ADMIN_USER_API_KEY }}
       cucumberPublishToken: ${{ secrets.IRS_CUCUMBER_PUBLISH_TOKEN }}
     with:
-      executionFilter: "!Ignore & !INACTIVE & INTEGRATION_TEST & DEV"
+      executionFilter: ${{ github.event.inputs.executionFilter }}
```

Comment:
AFAIK you need to set the default value here, in case the workflow is triggered by push to main

Reply:
the default is defined in inputs further above, if it is omitted the default filter `!Ignore & !INACTIVE & INTEGRATION_TEST & DEV` should be taken just as before, see [line 9](https://github.com/eclipse-tractusx/item-relationship-service/pull/735/files/b38a07228594449b6a25c9ead84a7e7b1ce8a0b8#diff-738f3bb681d2399174f75ab08289055a3e8da640bb9436ad09d6f4484c3e27a5R9)